core/src/test/java/org/elasticsearch/index/mapper/MultiFieldIncludeInAllMapperTests.java

```diff
@@ -0,0 +1,53 @@
+package org.elasticsearch.index.mapper;
+
+import org.elasticsearch.common.compress.CompressedXContent;
+import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.common.xcontent.XContentBuilder;
+import org.elasticsearch.index.MapperTestUtils;
+import org.elasticsearch.test.ESTestCase;
+
+import java.io.IOException;
+
+import static org.elasticsearch.common.xcontent.XContentFactory.jsonBuilder;
+import static org.hamcrest.core.IsEqual.equalTo;
+
+/**
+ * Created by makeyang on 2016/12/8.
+ */
+public class MultiFieldIncludeInAllMapperTests extends ESTestCase {
+    public void testExceptionForCopyToInMultiFields() throws IOException {
+        XContentBuilder mapping = createMappinmgWithIncludeInAllInMultiField();
+
+        // first check that for newer versions we throw exception if copy_to is found withing multi field
+        MapperService mapperService = MapperTestUtils.newMapperService(createTempDir(), Settings.EMPTY);
+        try {
+            mapperService.parse("type", new CompressedXContent(mapping.string()), true);
+            fail("Parsing should throw an exception because the mapping contains a include_in_all in a multi field");
```

Comment:
We tend to use `expectThrows` for this when we write new tests. It isn't required but it is a bit cleaner to read.